package/yast2-storage.spec

```diff
@@ -17,7 +17,7 @@
 
 
 Name:           yast2-storage
-Version:        3.2.16
+Version:        3.2.17
```

Comment:
Right, those clashing versions are a very valid point.

But probably we should have started numbering with 3.3.x for post-SP3.